srtcore/core.cpp

```diff
@@ -7117,6 +7389,8 @@ void CUDT::processCtrl(CPacket& ctrlpkt)
       // protect packet retransmission
       CGuard::enterCS(m_AckLock);
 
+      int32_t wrong_loss SRT_ATR_UNUSED = CSeqNo::m_iMaxSeqNo;
```

Comment:
Use `IF_HEAVY_LOGGING` macro.

Reply:
In this case it would be wrong because this variable is used with `LOGC`, not `HLOGC` macro.

Reply:
Aha, not it is LOGC.